lib/apr/views/commerce/commerce_order_slack_view.ex

```diff
@@ -170,6 +170,13 @@ defmodule Apr.Views.CommerceOrderSlackView do
     ]
   end
 
+  defp purchase_method(order) do
+    cond do
+      order["mode"] == "offer" && order["impulse_conversation_id"] -> "Inquiry Offer :cashmoney:"
+      true -> order["mode"]
```

Comment:
😄 (I don't know much about elixir but this line makes me happy)

Reply:
:/ I'm going to assume this is a fallback to the previous implementation